README.md

```diff
@@ -204,6 +225,13 @@ Building the Android APK is done most straightforwardly by using the pre-prepare
 
 To do so, run the following command for the 32bit docker
 
+```
```

Comment:
These are also ubuntuisms (or debian?), though for this we'd probably just say that explicitly here, and then add instructions for setting up docker on opensuse above it (just because that's the order they're in above). i can add those opensuse bits later, obviously don't expect you to be describing how to do that for a system you're not familiar with :D

Reply:
OS which used in docker is Ubuntu 18.04